lib/graph.ts

```diff
@@ -130,13 +139,19 @@ class GraphImpl implements Graph, Graph.Transformer {
 
   private _nodes: Node[];
 
-  constructor(graph: onnx.IGraphProto, graphInitializer?: Graph.Initializer) {
+  constructor(graph: onnx.IGraphProto|onnxruntime.experimental.fbs.Graph, graphInitializer?: Graph.Initializer) {
     if (!graph) {
       throw new TypeError('graph is empty');
     }
 
     // build the graph - will throw exceptions if something fatal is detected
-    this.buildGraph(graph);
+    if (graph instanceof onnx.GraphProto) {
+      this.buildGraph(graph);
```

Comment:
make buildGraph handle both onnx and ort, so we have buildGraph, buildGraphFromOnnxFormat, buildGraphFromOrtFormat?